www/_posts/2017-09-01-cordova-common-2.1.0.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+author:
+    name: Audrey So
+    url: https://twitter.com/aud_rey_so
+title:  "Cordova-Common@2.1.0 Released!"
+date: 2017-09-01
+categories: news
+tags: release tools cordova-common
+---
+
+We just released some changes to `cordova-common`!
+
+* [cordova@2.1.0](https://www.npmjs.com/package/cordova-common)
+
+Release Highlights:
+* Support added for `<config-file>` in `config.xml`
+* `allows-arbitrary-loads-for-media` attribute parsing added for `getAccesses`
+* Added variable replacing the `framework` tag
+* `JSON` uses 2 spaces for indentation
+
+To update your cordova CLI:
```

Comment:
Remove the to update part. It isn't really useful to consumers until after it has been included in next lib + platform releases

Reply:
Instead of the update, mention that `watch for this release to start rolling into upcoming platform and cordova-cli releases`